sklearn/tests/test_pipeline.py

```diff
@@ -418,6 +418,14 @@ def test_make_union():
     assert_equal(transformers, (pca, mock))
 
 
+def test_make_union_kwargs():
```

Comment:
maybe add a test that if an invalid ``**kwarg`` is given, there is an error? Otherwise LGTM.